plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -169,6 +171,33 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               });
             }
 
+            const containsServicePorts = portDefinitions.filter(function(
+              portDefinition
+            ) {
+              const result = portDefinition.servicePort
```

Comment:
`portDefinition.servicePort != null` checks both `null` and `undefined` please note `!=` not `!==`

Wheres `portDefinition.servicePort && portDefinition.servicePort !== 0` will return not strictly boolean value but Truthy and Falsy values instead.